### ISSUE TITLE
CodeCrunch indicating the usage of java.util.Objects

### ISSUE THREAD

**Issue by hiimss** [nus-cs2030/2122-s2#380]
# Summary
For some reason, CodeCrunch flagged me for using java.util.Objects when I did not import it (did not use java.util.* as well), does anybody know how to overcome this issue?

## Description 
Describe the problem and code snippets if necessary 

## Screenshots (if any): 
>
<img width="209" alt="Screenshot 2022-04-18 214014" src="https://user-images.githubusercontent.com/98826400/163816513-dd08d4dd-a879-4d9f-8413-1347f63bec9c.png">

**Comment by vinessa23** [comment]
maybe u use methods that are usually not allowed, like instanceof?

**Comment by jamietan123** [comment]
Yup, I think you need to check that you r not using methods from java.util.Objects

**Comment by alwinngjw** [comment]
Hi, yea i think you may have accidentally used methods from util.objects class. Might have to double check

**Comment by hiimss** [comment]
Hi everybody, thank you for the suggestions! Like what @junbin-tan said, we are unable to use anything from java.util.Objects regardless if we import it or not. If anybody is interested, I found the issue that caused this restriction, which was using :: in the map method (I just changed it to lambda instead).
